air/src/aux_table/hasher/mod.rs

```diff
@@ -8,13 +8,15 @@ use vm_core::{
     },
     utils::range as create_range,
 };
-use winter_air::TransitionConstraintDegree;
+use winter_air::{Assertion, TransitionConstraintDegree};
```

Comment:
nit: I would prefer to import both of these from `super`. (It looks like `TransitionConstraintDegree` was auto-imported and overlooked previously.)